demisto_sdk/commands/common/hook_validations/parsing_rule.py

```diff
@@ -65,3 +75,33 @@ def is_files_naming_correct(self):
                 self._is_valid = False
                 return False
         return True
+
+    def is_valid_rule_suffix_name(self):
+        """
+        Verifies the following::
+            1. The parsing rule file name ends with 'PARSING_RULE_FILE_SUFFIX'.
+            2. The parsing rule id ends with 'PARSING_RULE_ID_SUFFIX'.
+            3. The parsing rule name ends with 'PARSING_RULE_NAME_SUFFIX'.
+        """
+        with open(self.file_path) as yf:
+            data = yaml.load(yf)
+        rule_id = data.get("id", "")
+        rule_name = data.get("name", "")
+
+        invalid_suffix = {
+            "invalid_file_name": not re.search(
+                PARSING_RULE_FILE_SUFFIX_REGEX, self.file_path
+            ),
+            "invalid_id": not rule_id.endswith(PARSING_RULE_ID_SUFFIX),
+            "invalid_name": not rule_name.endswith(PARSING_RULE_NAME_SUFFIX),
+        }
+
+        if any(invalid_suffix.values()):
+            error_message, error_code = Errors.parsing_rule_suffix_name(
+                self.file_path, **invalid_suffix
+            )
+            if self.handle_error(error_message, error_code, file_path=self.file_path):
+                self._is_valid = False
+            return False
```

Comment:
this looks very similar to the one in modeling_rule, can we use the same code (with different arguments) for both?

Reply:
Yes. But there is no base rule validator or something like that to save this piece of code

Reply:
we can put it under `ContentEntityValidator`, and only use it in these cases here, or create a new file and put it there.